Retsinformation.js

```diff
@@ -0,0 +1,191 @@
+
+    ***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url){
+	if (ZU.xpathText(doc, '//div[@class="wrapper2"]/table[@id="ctl00_MainContent_ResultGrid1"]') && getSelectItems(doc, url).length) {
+		return "multiple";
+	} else if (url.indexOf("R0710") != -1) {
+		type = getType(doc, url);
+		return type;
+	} else return false;
+}
+
+function doWeb(doc, url) {
+	if (detectWeb(doc, url) == "multiple") {
+		var items = new Object();
+		var titles = getSelectItems(doc, url);
+		for (i = 0; i < titles.length; i++) {
+			items[titles[i].href] = ZU.trimInternal(titles[i].text);
+		}
+		Z.selectItems(items, function(selectedItems) {
+			if (!selectedItems) return true;
+			var urls = new Array();
+			for (var i in selectedItems) {
+				urls.push(i);
+			}
+			ZU.processDocuments(urls, scrape);
+		});
+	} else {
+		scrape(doc, url);
+	}
+}
+
+function getSelectItems(doc, url) {
+	if (/(R0900|R0920|R0930)/.test(url)) {
+		var titles = ZU.xpath(doc, '//table[@id="ctl00_MainContent_ResultGrid1"]/tbody/tr[@class!="th"]/td[1]/a[1]');
+	} else if (/(R0210|R0310)/.test(url)) {
+		var titles = ZU.xpath(doc, '//table[@id="ctl00_MainContent_ResultGrid1"]/tbody/tr[@class!="th"]/td[2]/a[1]');
+	} else if (/R0700.aspx\\?res/.test(url)) {
+		var titles = ZU.xpath(doc, '//table[@id="ctl00_MainContent_ResultGrid1"]/tbody/tr[@class!="th"]/td[3]/a[1]');
+	} else if (/(R0220|R0415|R0700)/.test(url)) {
+		var titles = ZU.xpath(doc, '//table[@id="ctl00_MainContent_ResultGrid1"]/tbody/tr[@class!="th"]/td[4]/a[1]');
+	}
+	return titles;
+}
+
+function scrape(doc, url) {
+	var type = getType(doc, url);
+	var newItem = new Zotero.Item(type);
+	newItem.title = getTitle(doc, url);
+	newItem.url = url;
+	if (type=="statute") {
+//		newItem.nameOfAct = newItem.title
+		newItem.codeNumber = getKortNavn(doc, url).id;
+		newItem.publicLawNumber = newItem.codeNumber;
+		newItem.dateEnacted = getKortNavn(doc, url).date;
+		newItem.shortTitle = getRessort(doc, url).shortTitle;
+		if (/L(BK|OV)/.test(newItem.publicLawNumber.substr(0,3))) {
+		} else {
+			newItem = addAuthor (newItem, getRessort(doc, url).ressort);
+		};
+	} else if (type=="case") {
+		newItem.caseName = newItem.title
+		newItem = addAuthor (newItem, getRessort(doc, url).ressort);		
+		newItem.docketNumber = getKortNavn(doc, url).id;
+		newItem.dateDecided = getKortNavn(doc, url).date;
+		newItem.shortTitle = getRessort(doc, url).shortTitle;
+	} else if (type=="report") {
+		newItem = addAuthor (newItem, getRessort(doc, url).ressort);
+		newItem.date = getRessort(doc, url).pubDate;
+		newItem.institution = "Folketinget";
+		threeLetters = getKortNavn(doc, url).threeLetters;
+		if (threeLetters=="EDP") {
+			newItem.seriesTitle = "Folketingets Ombudsmands driftundersøgelser";
+		} else if (threeLetters=="ISP") {
+			newItem.seriesTitle = "Folketingets Ombudsmands Inspektioner";
+		} else if (threeLetters=="FOU") {
+			newItem.seriesTitle = "Folketingets Ombudsmands udtalelser";
```

Comment:
Title capitalisation is odd, but it's taken directly from the webpage, e.g.: https://www.retsinformation.dk/Forms/R0415.aspx?res=104&nres=7